src/actix/auth.rs

```diff
@@ -5,7 +5,7 @@ use std::sync::Arc;
 use actix_web::body::{BoxBody, EitherBody};
 use actix_web::dev::{forward_ready, Service, ServiceRequest, ServiceResponse, Transform};
 use actix_web::http::Method;
-use actix_web::{Error, FromRequest, HttpMessage as _, HttpResponse, ResponseError};
```

Comment:
:pick: Nitpick: I write `use … as _` for traits to avoid polluting the namespace with unused identifiers. But I don't have a strong opinion about that.

Reply:
I think this was from some `fmt`. But my opinion is that it should not matter unless there are conflicts or a shorter name is more ergonomic.